packages/go_router/tool/run_tests.dart

```diff
@@ -9,25 +9,103 @@
 // ignore_for_file: avoid_print
 
 import 'dart:io';
+import 'package:io/io.dart' as io;
 import 'package:path/path.dart' as p;
+import 'package:yaml_edit/yaml_edit.dart';
 
 Future<void> main(List<String> args) async {
-  if (!Platform.isMacOS) {
-    print('This test can only be run on macOS.');
+  if (!Platform.isMacOS && !Platform.isWindows) {
```

Comment:
@chunhtai Is the intent to specifically exclude Linux for some reason? If so, there should be a comment explaining why.

Reply:
removed this to include all platforms